README.md

```diff
@@ -27,8 +27,8 @@ Tools built on DynamoRIO include:
   [drcpusim](http://dynamorio.org/docs/page_drcpusim.html)
 - The "strace for Windows" tool [drstrace](http://drmemory.org/strace_for_windows.html)
 - The code coverage tool [drcov](http://dynamorio.org/docs/page_drcov.html)
-- The library tracing tool [drltrace](http://drmemory.org/docs/page_drltrace.html)
```

Comment:
As you can see from the history, drltrace was recently moved to the drmemory repo and @mxmssh made this change to drmemory.org which is where the page will eventually be.  However the online docs have not yet been updated as they match release packages.  It's not clear when the next Dr. Memory release will be (too few people working on it...) so we'll have to decide what to do here.

If we had published nightly build packages (not yet added to Travis or Appveyor: #1967) we could point at the latest one of those?